Examples/Packages/manifest.json

```diff
@@ -9,15 +9,8 @@
     "com.mambojambostudios.unity-atoms-tags": "file:../../Packages/Tags",
     "com.mambojambostudios.unity-atoms-ui": "file:../../Packages/UI",
     "com.neuecc.unirx": "https://github.com/starikcetin/UniRx.git#7.1.0-upm",
-    "com.unity.2d.sprite": "1.0.0",
```

Comment:
Even though we support 2018.4, the Example project itself should run Unity 2019, and some of these packages are needed in that version. Please revert this file.